pkg/testcase/ingressdns.go

```diff
@@ -128,27 +128,27 @@ func validateIngressRoute(publicIp string) {
 	Eventually(func(g Gomega) {
 		pods, getErr := shared.GetPodsFiltered(filters)
 		g.Expect(getErr).NotTo(HaveOccurred(), getErr)
-		for _, pod := range pods {
-			g.Expect(pod.IP).NotTo(Equal("<none>"))
-			g.Expect(pod.IP).NotTo(BeEmpty())
-			if pod.IP != "<none>" && pod.IP != "" {
+		for i := range pods {
```

Comment:
I personally prefer the for loops using `for _, pod` 😕

Reply:
i prefer too but this one is a performance linter warning 
```
pkg/testcase/ingressdns.go:130:3: rangeValCopy: each iteration copies 160 bytes (consider pointers or indexing) (gocritic)
                for _, pod := range pods {
                ^
```

Reply:
Dangit gocritic! I'll accept it I guess